src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,10 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
+
+1.0.0b2
++++++++
+* Add --skip-gpu-driver-install option to node pool property in `az aks nodepool add`.
 * Add `--ip-families` to the `az aks update` command.
```

Comment:
This change is actually released in 1.0.0b1

Reply:
@FumingZhang do you want use to move the '--skip-gpu-driver-install' change to the section under 1.0.0b1?

Reply:
no

I mean the change "Add `--ip-families` to the `az aks update` command." is acutally released in `1.0.0b1`. Please help correct the history note by moving this to 1.0.0b1. You could keep the history note of your change in 1.0.0b2 and release a new version.

Reply:
@FumingZhang got you, I made the corresponding change according to your comment because @pineapplethevoyager  is OOF today.